src/es5processor.ts

```diff
@@ -284,6 +289,11 @@ class ES5Processor extends Rewriter {
     return this.isRequire(arg as ts.CallExpression);
   }
 
+  isEsModuleProperty(expr: ts.ExpressionStatement): boolean {
+    // We're matching the explicit source text generated by the TS compiler.
+    return expr.getText() === 'Object.defineProperty(exports, "__esModule", { value: true });';
```

Comment:
I considered it and am also on the fence but decided this was simpler.  I think if they change the output maybe it's worth having our tests notice it?